plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -93,6 +93,7 @@
     private final Set<String>                      commonMachineEnvVariables;
     private final String[]                         allMachinesExtraHosts;
     private final String                           projectFolderPath;
+    private final boolean                          locallySnapshot;
```

Comment:
do we need adjective or noun like localSnapshot
other booleans of this constructor are "doForce", "privilegeMode"